src/unity/python/turicreate/toolkits/classifier/logistic_classifier.py

```diff
@@ -621,10 +622,11 @@ class as a vector. The probability of the first class (sorted
         >>> class_predictions = model.predict(data, output_type='class')
 
         """
-
-        return super(_Classifier, self).predict(dataset,
-                                                output_type=output_type,
-                                                missing_value_action=missing_value_action)
+        if self.num_classes >2 and output_type=='probability':
+            raise ToolkitError("Not for multiclass")
```

Comment:
Could we add a more informative error message for users who may not understand what's going on?

Reply:
Yeah completely forgot to change it.